.github/workflows/chart-releaser.yaml

```diff
@@ -29,9 +29,9 @@ jobs:
           helm repo update
 
       - name: Run chart-releaser
-        uses: helm/chart-releaser-action@v1.2.1
+        uses: helm/chart-releaser-action@v1.5.0
         env:
-          CR_TOKEN: "${{ secrets.BROADBOT_TOKEN }}"
+          CR_TOKEN: "${{ secrets.GITHUB_TOKEN }}"
```

Comment:
I think we also need to make these changes in releasedr.yaml

Reply:
Good point!